### ISSUE TITLE
Cheetah licensing

### ISSUE THREAD

**Issue by zSeriesGuy** [Picovoice/cheetah#16]
I developed a speech-to-text application for sharing the transcript to many concurrently.
See [https://github.com/zSeriesGuy/SpeakReader](https://github.com/zSeriesGuy/SpeakReader)

It takes microphone or line-in input to the server and sends the transcription to any number of connected web browsers.
It currently uses the Google Speech-to-text cloud platform. But I intend to extend it to support other speech-to-text engines.
I found Cheetah to be an interesting option.

How does the licensing work? My project requires the user of the app to set up their own license with Google Cloud. If I were to incorporate Cheetah as a selectable stt engine option, how would someone obtain a license file?

**Comment by kenarsa** [comment]
is this commercial? if not, then you can use whats here.

**Comment by zSeriesGuy** [comment]
My project is not a commercial product. But it requires a speech-to-text engine.
As it is currently written, the user of the app would have to buy cloud services.
My project is python-based and can be run on any platform supporting python 3.
Incorporating the free version of Cheetah then doesn't seem practical since it would then limit the user to a linux platform.
In addition, I saw in the answer to issue #15 that you don't currently plan to sell any kind of individual license.
I would need the user of my application to purchase a license from you for their use.

I saw in the free version that there is a time limit of 100 seconds. What is the 100 seconds? Over what time frame?
My app uses a streaming generator to Google speech-to-text service. I currently run the app for 1 hour each Sunday. How would this work with Cheetah?
